install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2646,6 +2646,11 @@ auth:
   mTLS:
     # -- Enable mtls-spiffe authentication method in CiliumNetworkPolicy
     enabled: false
+    # -- SPIRE server endpoint
```

Comment:
These are just temporary, right? As in, once we manage spire, these knobs are not to be touched by end-users?

If so, then perhaps we shouldn't even document them.

Reply:
yes, this is just temporary.

Reply:
But it might be used for expert users who need an external spire server in the future

Reply:
Temporary until when? If they make it to 1.14 and we want to remove them we'll need to go through deprecation etc. Are these values required to be in the Helm chart if it's temporary? Would it make sense to add a comment to warn they will be removed?

(That is, unless we decide to keep them, as Maartje suggests, of course.)

Reply:
the SPIFEE installation work is currently in progress, so I reckon we will decide to clean up or keep this option very soon, probably snapshot.2.

Reply:
Yes, we will have this tidied up one way or another in snapshot.2.